manager/config/integration-tests/deployment_configmap.yaml

```diff
@@ -24,11 +24,11 @@ data:
   #EXTENSIONS_POLICY_MANAGER_NAME: "OPA"
 
   #internal vault related variables
-  VAULT_ADDRESS: "http://vault.m4d-system:8200/"
+  VAULT_ADDRESS: "http://vault.m4d-system:8200"
   VAULT_DATASET_HOME: "m4d/dataset-creds/"
   VAULT_USER_HOME: "m4d/user-creds/"
-  VAULT_DATASET_MOUNT: "v1/sys/mounts/m4d/dataset-creds"
-  VAULT_USER_MOUNT: "v1/sys/mounts/m4d/user-creds"
+  VAULT_DATASET_MOUNT: "/v1/sys/mounts/m4d/dataset-creds"
```

Comment:
I think you also have to adapt these settings in charts/m4d/templates/m4d-config.yaml

Reply:
Fixed.